docs/rootfs-and-kernel-setup.md

```diff
@@ -47,6 +49,23 @@ can boot:
 1. Upon a successful build, you can find the uncompressed kernel image under
    `./vmlinux`.
 
+### Use the provided recipe
+
+The kernel images used in our CI to test Firecracker's features are obtained by
+using the recipe inside devtool:
+
+```bash
+./tools/devtool build_kernel -c ./resources/microvm-kernel-x86.config -n 8
+```
+
+or
+
+```bash
+./tools/devtool build_kernel -c ./resources/microvm-kernel-arm64.config -n 8
```

Comment:
nit: it'd be even easier to use if we added the paths to the configs as default parameters. wdyt?

Reply:
Indeed i thought about adding a default location for the config but I gave up the idea. My skepticism comes from the fact that we are not (as of yet) running this recipe with every PR and as a consequence, we run the risk of breaking it if the config file gets renamed or relocated. If you still think it would add value to define default paths to the config, let me know/